core/templates/pages/contributor-dashboard-page/question-opportunities/question-opportunities.component.ts

```diff
@@ -137,7 +138,7 @@ angular.module('oppia').component('questionOpportunities', {
           QuestionObjectFactory.createDefaultQuestion([skillId]);
         const questionId = question.getId();
         const questionStateData = question.getStateData();
-
+        ContextService.setImageSaveDestinationToLocalStorage();
```

Comment:
Should we do this in the modal controller file? As the question creation is controlled there, right?

Reply:
done, moved it to controller